<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="KTLPT1386" failures="0" tests="23" name="APITestCase.ManageLead" time="9.680" errors="1" timestamp="2024-09-04T12:53:45 IST" skipped="1">
  <testcase classname="APITestCase.ManageLead" name="UploadLeadAttachment" time="0.268"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="ExportLeads" time="3.533"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetLeadDetails" time="0.161"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="MarkLeadAsStar" time="0.153"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetLeadActivity" time="0.108"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="RescheduleTask" time="0.172"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="UpdateLeadStage" time="0.000">
    <skipped/>
  </testcase> <!-- UpdateLeadStage -->
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="AddTaskForLead" time="0.200"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="AddLeadActivity" time="0.293"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="MarkTaskAsCompleted" time="0.157"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="DeleteAttachment" time="0.157"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetLeadOwner" time="0.201"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetTaskByLeadId" time="0.131"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetCity" time="0.126"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="AddLead" time="0.495">
    <error message="Cannot invoke &quot;java.lang.Integer.intValue()&quot; because the return value of &quot;io.restassured.path.json.JsonPath.get(String)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "io.restassured.path.json.JsonPath.get(String)" is null
at APITestCase.ManageLead.AddLead(ManageLead.java:97)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- AddLead -->
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="UpdateLead" time="0.305"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetLeadHistory" time="0.170"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetRequirement" time="0.114"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetLeadTask" time="0.176"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetLeads" time="2.395"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetStates" time="0.112"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="GetLeadStage" time="0.111"/>
  <system-out/>
  <testcase classname="APITestCase.ManageLead" name="CheckLeadNumbers" time="0.142"/>
  <system-out/>
</testsuite> <!-- APITestCase.ManageLead -->
